scripts/generate_dump

```diff
@@ -6,6 +6,8 @@
 
 set -u
 
+ERROR_INTERRUPTED=1
+ERROR_TERMINATED=2
```

Comment:
do you have any idea why the error codes didnt start from 1 in the first place and they got strange numbers like 6 and 10?

Reply:
In the recent utilities version in master it is fixed by Vivek. After rebase I addopted my changes.